acceptance-tests/features/register.feature

```diff
@@ -18,17 +18,20 @@ Feature: Users can sign up to the self-service experience
 
   Rule: Users can only register with email addresses from certain domains
     Scenario Outline: User can register allowed email addresses
-
       When they submit the email <email>
       Then they should be redirected to the "/register/enter-email-code" page
 
       Examples:
         | email                                     |
-        | "test-user@test.gov.uk" |
-        | "test-user@digital.cabinet-office.gov.uk"      |
+        | "test-user@test.gov.uk"                   |
```

Comment:
See comment below re allowed test domains

Reply:
as mentioned below to cover all the supported ones I have added the supported list